platform/mellanox/sdk.mk

```diff
@@ -17,6 +17,7 @@
 MLNX_SDK_VERSION = 4.5.4206
 MLNX_SDK_ISSU_VERSION = 101
 
+MLNX_SDK_DRIVERS_GITHUB_URL = https://github.com/Mellanox/Spectrum-SDK-Drivers
```

Comment:
>MLNX_SDK_DRIVERS_GITHUB_URL [](http://example.com/codeflow?start=0&length=27)

Could you reuse this in MLNX_SX_KERNEL_GITHUB_URL_BASE?

Reply:
Actually, i can use MLNX_SX_KERNEL_GITHUB_URL_BASE directly in my code. no need for a different variable. Thanks for noticing

Reply:
Sry, you original suggestion is preferable. I've updated it